### DISCUSSION TITLE
blog/papermill-execute-notebook/

### DISCUSSION THREAD

**Discussion by giscus** [jeremy-feng/giscus#16] [Announcements]
# blog/papermill-execute-notebook/

https://fengchao.pro/blog/papermill-execute-notebook/

**Comment by Zachary722** [comment]
又学会一招，顺便催更学长(#^.^#)

**Reply by jeremy-feng** [reply]
哈哈谢谢关注~（方便私一下是谁么hhh :nerd_face:
